### DISCUSSION TITLE
circle at lower-right corner is huge

### DISCUSSION THREAD

**Discussion by kenfhill84083** [freeplane/freeplane#1223] [Help] [answered]
When I hover over a node that has children I see a huge circle in the lower-right corner of the node - indicating children I assume.  It is so big it partially covers a child node.  Is there any way to control the size of this circle?

![big fold circle](https://github.com/freeplane/freeplane/assets/6380301/5aa69557-f23e-4ceb-8204-886ed5dca434)

**Comment by dpolivaev** [comment, marked as answer]
Already available in 1.11.5_05, Preferences…->Appearance->Size limits->Folding switch width

**Reply by kenfhill84083** [reply]
> Already available in 1.11.5_05, Preferences…->Appearance->Size limits->Folding switch width

I could not find "Folding switch width" in command search.   Folding symbol width does not seem to affect my issue.
![preferences - size limits](https://github.com/freeplane/freeplane/assets/6380301/a5a8216b-a507-47af-b62f-71173884da41)

I installed that version earlier this morning.
![freeplane setup 1 11 5 pre 05](https://github.com/freeplane/freeplane/assets/6380301/c26d403a-6a80-4a2d-ab74-2d76d70b5688)

**Reply by dpolivaev** [reply]
I am sorry, maybe I have added it later. I have just released pre06, look there.

**Reply by kenfhill84083** [reply]
> I am sorry, maybe I have added it later. I have just released pre06, look there.

That looks much better.  Thanks.

**Reply by abc16361** [reply]
Pre06 works very well. Thank you!
